learningresources/models.py

```diff
@@ -88,5 +92,20 @@ def save(self, *args, **kwargs):
                 slug = "{0}{1}".format(slugify(self.name), count)
                 count += 1
             self.slug = slug
+            is_create = True
```

Comment:
This isn't strictly true, I think you want to just check self.id is None for this value. The rest of the expression (self.name != get_obj... ) is there to deal with updating slugs if the name changes for an updated repository